### ISSUE TITLE
Migration to version 17.0

### ISSUE THREAD

**Issue by legalsylvain** [OCA/OpenUpgrade#4124]
### Important note : 
- Some modules are done and PR has been proposed. Please review first the following [PRs that are not blocked by dependencies, here.](https://github.com/OCA/OpenUpgrade/pulls?q=is%3Apr+is%3Aopen+label%3A%22Dependency+OK%22+milestone%3A17.0+)

- Full list of modules available here : https://github.com/OCA/OpenUpgrade/blob/17.0/docsource/modules160-170.rst 

# Pending / done work

- [x] account - By @hbrunn - #4668
- [x] account_check_printing - By @CarlosRoca13 - #4778
- [x] account_debit_note - By @carlos-lopez-tecnativa - #5415
- [x] account_edi - By @carlos-lopez-tecnativa - #4771
- [x] account_edi_ubl_cii - By @MiquelRForgeFlow - #4812
- [x] account_fleet - By @carlos-lopez-tecnativa - #5416
- [x] account_payment - By @mgomezleon - #4650
- [x] account_payment_term - By @MiquelRForgeFlow - #4851
- [x] account_qr_code_sepa - By @acpMicrocom - #4607
- [x] analytic - By @duong77476-viindoo / @hbrunn / @pedrobaeza - #4597
- [x] association - By @pedrobaeza - #4933
- [x] attachment_indexation - By @acpMicrocom - #4608
- [x] auth_ldap - By @carlos-lopez-tecnativa - #5412
- [x] auth_oauth - By @carlos-lopez-tecnativa - #4767
- [x] auth_password_policy - By @carlos-lopez-tecnativa - #4768
- [x] auth_password_policy_signup - By @carlos-lopez-tecnativa - #4769
- [x] auth_signup - By @flachica - #4487
- [x] auth_totp - By @legalsylvain - #4635
- [x] auth_totp_mail - By @MiquelRForgeFlow - #4850
- [x] auth_totp_portal - By @acpMicrocom - #4609
- [x] barcodes - By @hbrunn - #4690
- [x] barcodes_gs1_nomenclature - By @hbrunn - #4690
- [x] base - By @duong77476-viindoo / @pedrobaeza - #4582
- [x] base_address_extended - By @pilarvargas-tecnativa - #4700
- [x] base_automation - By @petrus-v - #5028
- [x] base_geolocalize - By @acpMicrocom - #4588
- [x] base_iban - By @acpMicrocom - #4610
- [x] base_import - By @acpMicrocom - #4611
- [x] base_import_module - By @legalsylvain - #4636
- [x] base_install_request - By @MiquelRForgeFlow - #4849
- [x] base_setup - By @acpMicrocom - #4612
- [x] base_sparse_field - By @CarlosRoca13 - #4780
- [x] base_vat - By @carlos-lopez-tecnativa - #4766
- [x] board - By @carlos-lopez-tecnativa - #4770
- [x] bus - By @pedrobaeza - #4631
- [x] calendar - By @duong77476-viindoo - #4460
- [x] contacts - By @acpMicrocom - #4603
- [x] crm - By @acpMicrocom - #4554
- [x] crm_iap_mine - By @ivantodorovich - #5110
- [x] crm_livechat - By @carlos-lopez-tecnativa - #5068
- [x] crm_sms - By @acpMicrocom - #4589
- [x] delivery - By @MiquelRForgeFlow - #4751
- [x] digest - By @hbrunn / @pedrobaeza - #4642
- [x] event - By @victoralmau - #4663
- [x] event_crm - By @carlos-lopez-tecnativa - #4887
- [x] event_sale - By @pedrobaeza - #4936
- [x] fleet - By @victoralmau - #4666
- [x] gamification - By @MiquelRForgeFlow - #4717
- [x] gamification_sale_crm - By @acpMicrocom - #4613
- [x] google_account - By @pedrobaeza - #4830
- [x] google_calendar - By @carlos-lopez-tecnativa - #4859
- [x] google_gmail - By @pedrobaeza - #4831
- [x] google_recaptcha - By @acpMicrocom - #4614
- [x] hr - By @pedrobaeza - #4794
- [x] hr_attendance - By @MiquelRForgeFlow - #4801
- [x] hr_contract - By @duong77476-viindoo - #4463
- [x] hr_expense - By @MiquelRForgeFlow - #4826
- [x] hr_fleet - By @MiquelRForgeFlow - #5112
- [x] hr_gamification - By @carlos-lopez-tecnativa - #4863
- [x] hr_holidays - By @remi-filament - #4702
- [x] hr_holidays_attendance - By @carlos-lopez-tecnativa - #4866
- [x] hr_hourly_cost - By @carlos-lopez-tecnativa - #4888
- [x] hr_maintenance - By @victoralmau - #4658
- [x] hr_org_chart - By @acpMicrocom - #4590
- [x] hr_recruitment - By @remi-filament - #4703
- [x] hr_recruitment_skills - By @carlos-lopez-tecnativa - #5303
- [x] hr_recruitment_survey - By @carlos-lopez-tecnativa - #4919
- [x] hr_skills - By @pedrobaeza - #5205
- [x] hr_skills_slides - By @pedrobaeza - #5204
- [x] hr_skills_survey - By @pedrobaeza - #5206
- [x] hr_timesheet - By @pedrobaeza - #4908
- [x] hr_timesheet_attendance - By @carlos-lopez-tecnativa - #4913
- [x] hr_work_entry - By @remi-filament - #5011
- [x] hr_work_entry_contract - By @remi-filament - #5012
- [x] hr_work_entry_holidays - By @remi-filament - #5013
- [x] http_routing - By @acpMicrocom - #4626
- [x] iap - By @legalsylvain - #4689
- [x] iap_crm - By @MiquelRForgeFlow - #5111
- [x] iap_mail - By @acpMicrocom - #4615
- [x] im_livechat - By @MiquelRForgeFlow - #4715
- [x] l10n_be - By @pedrobaeza - #5116
- [x] l10n_co - By @david-banon-tecnativa - #5530
- [x] l10n_de - By @pedrobaeza - #4990
- [x] l10n_de_skr03 - By @pedrobaeza - #4994
- [x] l10n_de_skr04 - By @pedrobaeza - #4994
- [x] l10n_dk - By @MiquelRForgeFlow - #5184
- [x] l10n_es - By @CarlosRoca13 - #4795
- [x] l10n_generic_coa - By @pilarvargas-tecnativa - #4698
- [x] l10n_latam_base - By @carlos-lopez-tecnativa - #5413
- [x] l10n_latam_invoice_document - By @carlos-lopez-tecnativa - #5414
- [x] l10n_multilang - By @pedrobaeza - #5115
- [ ] l10n_mx - By @david-banon-tecnativa - #5534
- [x] l10n_pe - By @david-banon-tecnativa - #5532
- [x] l10n_pt - By @pedrobaeza - #5478
- [ ] l10n_vn - By @duong77476-viindoo - #4464
- [x] link_tracker - By @MiquelRForgeFlow - #4714
- [x] loyalty - By @pilarvargas-tecnativa - #5250
- [x] mail - By @hbrunn - #4600
- [x] mail_bot - By @MiquelRForgeFlow - #4817
- [x] mail_bot_hr - By @acpMicrocom - #4616
- [x] maintenance - By @victoralmau - #4654
- [x] maintenance_plan_activity - By @victoralmau - #4752
- [x] mass_mailing - By @carlos-lopez-tecnativa - #4785
- [x] mass_mailing_crm - By @carlos-lopez-tecnativa - #4861
- [x] mass_mailing_event - By @carlos-lopez-tecnativa - #4890
- [x] mass_mailing_event_track - By @carlos-lopez-tecnativa - #4891
- [x] mass_mailing_sale - By @carlos-lopez-tecnativa - #4860
- [x] mass_mailing_slides - By @acpMicrocom - #4617
- [x] membership - By @MiquelRForgeFlow - #4835
- [x] microsoft_account - By @carlos-lopez-tecnativa - #5065
- [x] microsoft_calendar - By @carlos-lopez-tecnativa - #5066
- [x] microsoft_outlook - By @acpMicrocom - #4618
- [x] mrp - By @pedrobaeza - #4843
- [x] mrp_account - By @pedrobaeza - #4856
- [x] mrp_landed_costs - By @pedrobaeza - #4846
- [x] note - By @ndd-odoo / @pedrobaeza - #4914
- [x] onboarding - By @duong77476-viindoo - #4465
- [x] partner_autocomplete - By @acpMicrocom - #4628
- [x] payment - By @hbrunn - #4692
- [x] payment_custom - By @remi-filament - #4592
- [x] payment_paypal - By @carlos-lopez-tecnativa - #4996
- [x] payment_stripe - By @pedrobaeza - #4935
- [x] phone_validation - By @acpMicrocom - #4627
- [x] point_of_sale - By @flachica - #4489
- [x] portal - By @duong77476-viindoo - #4467
- [x] portal_rating - By @pilarvargas-tecnativa - #4777
- [x] pos_discount - By @david-banon-tecnativa - #5514
- [x] pos_epson_printer - By @carlos-lopez-tecnativa - #5315
- [x] pos_epson_printer_restaurant - By @carlos-lopez-tecnativa - #5327
- [x] pos_hr - By @carlos-lopez-tecnativa - #5304
- [x] pos_hr_restaurant - By @carlos-lopez-tecnativa - #5316
- [x] pos_mrp - By @carlos-lopez-tecnativa - #5329
- [x] pos_restaurant - By @carlos-lopez-tecnativa - #5313
- [x] pos_sale - By @carlos-lopez-tecnativa - #5311
- [ ] pos_sale_product_configurator - By @pedrobaeza - #5456
- [x] privacy_lookup - By @MiquelRForgeFlow - #4848
- [x] product - By @duong77476-viindoo / @hbrunn / @pedrobaeza - #4598
- [x] product_email_template - By @david-banon-tecnativa - #5512
- [x] product_expiry - By @CarlosRoca13 - #4782
- [x] product_margin - By @carlos-lopez-tecnativa - #4974
- [x] product_matrix - By @MiquelRForgeFlow - #4815
- [x] project - By @pedrobaeza - #4892
- [x] project_account - By @carlos-lopez-tecnativa - #4973
- [x] project_hr_expense - By @MiquelRForgeFlow - #5616
- [x] project_purchase - By @carlos-lopez-tecnativa - #4905
- [x] project_sale_expense - By @MiquelRForgeFlow - #5616
- [x] project_timesheet_holidays - By @remi-filament - #5010
- [x] project_todo - By @ndd-odoo / @pedrobaeza - #4914
- [x] purchase - By @hbrunn - #4699
- [x] purchase_mrp - By @MiquelRForgeFlow - #4854
- [x] purchase_product_matrix - By @MiquelRForgeFlow - #4815
- [x] purchase_requisition - By @acpMicrocom - #4619
- [x] purchase_requisition_stock - By @MiquelRForgeFlow - #4814
- [x] purchase_stock - By @MiquelRForgeFlow - #4746 / By @pedrobaeza - #4756 
- [x] rating - By @acpMicrocom - #4643
- [x] repair - By @carlos-lopez-tecnativa - #5044
- [x] resource - By @duong77476-viindoo - #4471
- [x] sale - By @hbrunn - #4691
- [x] sale_async_emails - By @MiquelRForgeFlow - #4853
- [x] sale_crm - By @MiquelRForgeFlow - #4754
- [x] sale_expense - By @MiquelRForgeFlow - #4862
- [x] sale_loyalty - By @pilarvargas-tecnativa - #5267
- [x] sale_loyalty_delivery - By @pilarvargas-tecnativa - #5269
- [x] sale_management - By @pedrobaeza - #4757
- [x] sale_margin - By @carlos-lopez-tecnativa - #4772
- [x] sale_mrp - By @pedrobaeza - #4845
- [x] sale_product_configurator - By @acpMicrocom - #4648
- [x] sale_project - By @remi-filament - #4704
- [x] sale_project_stock - By @MiquelRForgeFlow - #5616
- [x] sale_purchase - By @MiquelRForgeFlow - #4755
- [x] sale_purchase_stock - By @acpMicrocom - #4621
- [x] sale_stock - By @pedrobaeza - #4749
- [x] sale_stock_margin - By @carlos-lopez-tecnativa - #5328
- [x] sale_timesheet - By @carlos-lopez-tecnativa - #4910
- [x] sales_team - By @hbrunn - #4685
- [x] sms - By @acpMicrocom - #4622
- [x] snailmail - By @acpMicrocom - #4646
- [x] snailmail_account - By @acpMicrocom - #4623
- [x] social_media - By @MiquelRForgeFlow - #4753
- [x] spreadsheet - By @legalsylvain - #4641
- [x] spreadsheet_account - By @MiquelRForgeFlow - #4858
- [x] spreadsheet_dashboard - By @MiquelRForgeFlow - #4857
- [x] spreadsheet_dashboard_event_sale - By @pedrobaeza - #4937
- [x] spreadsheet_dashboard_hr_timesheet - By @pedrobaeza - #4907
- [x] spreadsheet_dashboard_im_livechat - By @pedrobaeza - #5351
- [x] spreadsheet_dashboard_pos_hr - By @pedrobaeza - #5454
- [x] spreadsheet_dashboard_purchase - By @MiquelRForgeFlow - #4864
- [x] spreadsheet_dashboard_purchase_stock - By @MiquelRForgeFlow - #4864
- [x] spreadsheet_dashboard_sale_expense - By @MiquelRForgeFlow - #4864
- [x] spreadsheet_dashboard_sale_timesheet - By @pedrobaeza - #4886
- [x] spreadsheet_dashboard_stock_account - By @MiquelRForgeFlow - #4864
- [x] stock - By @hbrunn - #4681
- [x] stock_account - By @MiquelRForgeFlow - #4733
- [x] stock_delivery - By @MiquelRForgeFlow - #4751
- [x] stock_dropshipping - By @pedrobaeza - #4829
- [x] stock_landed_costs - By @pedrobaeza - #4828
- [x] stock_picking_batch - By @CarlosRoca13 - #4779
- [x] survey - By @MiquelRForgeFlow - #4818
- [x] theme_default - By @pedrobaeza - #5008
- [x] uom - By @legalsylvain - #4640
- [x] utm - By @victoralmau - #4664
- [x] web - By @remi-filament - #4583
- [x] web_editor - By @legalsylvain - #4638
- [x] web_hierarchy - By @MiquelRForgeFlow - #4810
- [x] web_tour - By @pedrobaeza - #4632
- [x] web_unsplash - By @acpMicrocom - #4624
- [x] website - By @MiquelRForgeFlow - #4821
- [x] website_blog - By @pilarvargas-tecnativa - #4797
- [x] website_crm - By @pilarvargas-tecnativa - #4776
- [x] website_crm_livechat - By @carlos-lopez-tecnativa - #5330
- [x] website_crm_sms - By @acpMicrocom - #4604
- [x] website_event - By @pilarvargas-tecnativa - #4676
- [x] website_event_sale - By @pilarvargas-tecnativa - #4677
- [x] website_event_track - By @carlos-lopez-tecnativa - #4889
- [x] website_form_project - By @carlos-lopez-tecnativa - #4906
- [x] website_forum - By @pedrobaeza - #4938
- [x] website_google_map - By @MiquelRForgeFlow - #4834
- [x] website_hr_recruitment - By @carlos-lopez-tecnativa - #4912
- [x] website_links - By @carlos-lopez-tecnativa - #4865
- [x] website_livechat - By @carlos-lopez-tecnativa - #5070
- [x] website_mail - By @acpMicrocom - #4645
- [x] website_mass_mailing - By @pilarvargas-tecnativa - #4799
- [x] website_membership - By @pilarvargas-tecnativa - #4678
- [x] website_partner - By @pilarvargas-tecnativa - #4688
- [x] website_payment - By @pilarvargas-tecnativa - #4798
- [x] website_profile - By @MiquelRForgeFlow - #4833
- [x] website_sale - By @pilarvargas-tecnativa - #4687
- [x] website_sale_comparison - By @acpMicrocom - #4629
- [x] website_sale_delivery - By @carlos-lopez-tecnativa - #4975
- [x] website_sale_digital - By @MiquelRForgeFlow - #4879
- [x] website_sale_loyalty - By @pilarvargas-tecnativa - #5268
- [x] website_sale_picking - By @carlos-lopez-tecnativa - #5331
- [x] website_sale_product_configurator - By @carlos-lopez-tecnativa - #5417
- [x] website_sale_slides - By @carlos-lopez-tecnativa - #4983
- [x] website_sale_stock - By @pilarvargas-tecnativa - #4775
- [x] website_sale_wishlist - By @acpMicrocom - #4630
- [x] website_slides - By @MiquelRForgeFlow - #4838
- [x] website_slides_forum - By @pedrobaeza - #5052
- [x] website_slides_survey - By @carlos-lopez-tecnativa - #4984
- [x] website_sms - By @acpMicrocom - #4605
- [x] website_twitter - By @carlos-lopez-tecnativa - #5069

**Not understood removal**

- ``spreadsheet_dashboard_sale_expense`` : ???
- ``l10n_ae_pos`` : ???
- ``l10n_sa_edi_pos`` : ???

- ``website_sale_digital`` https://github.com/odoo/odoo/pull/132739 (removal or merge ?)

**Renamed modules**

- ``note`` renamed into ``project_todo`` in https://github.com/odoo/odoo/pull/115390
- ``website_sale_mondialrelay`` renamed into ``website_sale_delivery_mondialrelay`` in https://github.com/odoo/odoo/pull/110686

**Merged modules** 

- ``account_payment_invoice_online_payment_patch`` merged into ``account_payment`` in https://github.com/odoo/odoo/pull/111046
- ``account_sequence`` merged into ``account`` in https://github.com/odoo/odoo/pull/106413
- ``association`` merged into ``membership`` in https://github.com/odoo/odoo/pull/106472
- ``l10n_de_skr03`` merged into ``l10n_de`` in https://github.com/odoo/odoo/pull/116274
- ``l10n_de_skr04`` merged into ``l10n_de`` in https://github.com/odoo/odoo/pull/116274
- ``l10n_hr_euro`` merged into ``l10n_hr`` in https://github.com/odoo/odoo/pull/108511
- ``l10n_in_tcs_tds`` merged into ``l10n_in`` in https://github.com/odoo/odoo/pull/110016
- ``l10n_in_upi`` merged into ``l10n_in`` in https://github.com/odoo/odoo/pull/111299
- ``l10n_latam_account_sequence`` merged ``into l10n_latam_invoice_document`` : https://github.com/odoo/odoo/pull/106413
- ``loyalty_delivery`` merged into  ``sale_loyalty_delivery`` in https://github.com/odoo/odoo/pull/129572
- ``pos_cache`` merged into ``point_of_sale`` in https://github.com/odoo/odoo/pull/104423
- ``pos_daily_sales_reports`` merged into ``point_of_sale`` in https://github.com/odoo/odoo/pull/113380
- ``pos_epson_printer_restaurant`` merged into ``point_of_sale`` in https://github.com/odoo/odoo/pull/114655
- ``purchase_price_diff`` merged into ``purchase_stock`` in https://github.com/odoo/odoo/pull/109924

- ``web_kanban_gauge`` merged into ``web`` in https://github.com/odoo/odoo/pull/115293 
- ``website_event_crm_questions`` merged into ``website_event_crm`` in https://github.com/odoo/odoo/pull/112164
- ``website_sale_delivery`` merged into ``website_sale`` in https://github.com/odoo/odoo/pull/110686

- ``website_sale_loyalty_delivery`` merged into ``website_sale_loyalty`` in https://github.com/odoo/odoo/pull/110686
- ``website_sale_stock_product_configurator`` merged into ``website_sale_product_configurator`` in https://github.com/odoo/odoo/pull/125960

**Removed modules**

- ``l10n_lu_peppol_id`` removed in https://github.com/odoo/odoo/pull/115934
- ``l10n_multilang`` removed in https://github.com/odoo/odoo/pull/110016
- ``website_payment_paypal`` removed in https://github.com/odoo/odoo/pull/113138

**Changes to be taken into account**

- https://github.com/odoo/odoo/pull/124182
- [x] openupgrade_framework - By @royle-vietnam - #4427
- [x] openupgrade_scripts - By @StefanRijnhart - #4294

**Comment by legalsylvain** [comment]
hi @OCA/openupgrade-maintainers. 

version 17 will be released in less than 2 months. Promotion began at odoo community days in India last week.
For the moment, the master branch is somewhere between version saas-16.4 and 17.0.

here is a first analysis of the changes regarding modules, between 16.0 and 17.0 could you take a look ? This can be used as a basis for building the next ``apriori.py`` file.

Note : in the first section I didn't found some removals. For exemple, ``git log --full-history -- addons/l10n_sa_edi_pos/__manifest__.py`` return nothing. So I don't know when the module disappeared, and why. Help is so welcome.

Thanks ! 

CC : @remytms, @carmenbianca, @pedrobaeza, @MiquelRForgeFlow, @StefanRijnhart

**Comment by legalsylvain** [comment]
FYI, working on initializing 17.0 branch.

**Comment by danieledraganti** [comment]
Hello; just wondering, what is the status on OpenUpgrade for 17.0? I see the 17.0 branch is quite behind compared to 16.0.

**Comment by legalsylvain** [comment]
hi @danieledraganti, thanks for your interest.

AFAIK, No work done for the time being. I finally didn't have time to initialize the project.

**Comment by danieledraganti** [comment]
@legalsylvain Oh, okay. Thank you for the feedback!

**Comment by tracencode** [comment]
Hello @legalsylvain @pedrobaeza , is there any specific date for openupgrade-17.0 release?

**Comment by TVerberne** [comment]
Hi @legalsylvain 
Any news on 17? Just found out it's not official yet but I see your FYI post that was posted already in November last year.
Thanks

**Comment by JordiBForgeFlow** [comment]
@TVerberne @tracencode You can start with the migration yourself, and provide us the feedback as you make progress. You are expected to contribute, as much as existing contributors.

**Comment by TVerberne** [comment]
@JordiBForgeFlow I am not an Odoo dev unfortunately, just looking for a way to upgrade Odoo from 15 to 17 atm :) Like most of us that found this issue I assume.

**Comment by JordiBForgeFlow** [comment]
@TVerberne Then you can possibly contact one of the contributors and pay them to fund the migration.

**Comment by pedrobaeza** [comment]
To take into account for noupdate_changes: https://github.com/odoo/odoo/commit/b82a1ae38b0eb72bdf39f32a87b6431a91628939

**Comment by daiduongnguyen-odoo** [comment]
> To take into account for noupdate_changes: [odoo/odoo@b82a1ae](https://github.com/odoo/odoo/commit/b82a1ae38b0eb72bdf39f32a87b6431a91628939)

Done in https://github.com/OCA/OpenUpgrade/pull/4428

**Comment by janiniravc** [comment]
Hello Team,

We are currently experiencing an issue during the migration process from version 16 to version 17 of Odoo. the following error message: "error: option --upgrade-path: the path './PATH_TO_openupgrade_scripts_MODULE/scripts/' is not a valid upgrade directory"

We kindly request your assistance in resolving this matter.

Thank you

**Comment by pedrobaeza** [comment]
@janiniravc use Discussions for this. Anyway, basically you can't migrate to 17 yet.

**Comment by sunarcrajneesh** [comment]
I recently came across the "website_sale_checkout_skip_payment" module available for Odoo 16 in the OCA e-commerce repository (link: https://github.com/OCA/e-commerce/tree/16.0/website_sale_checkout_skip_payment).

I was wondering if there are any plans to update this module for Odoo 17. This functionality would be very useful for our project, and I would appreciate it if you could let us know if there's any ongoing work or if there are alternative solutions available for Odoo 17.

Thank you for your time and effort in maintaining this valuable module, and I look forward to hearing from you.

**Comment by prakashjain** [comment]
I recently came across the "website_sale_checkout_skip_payment" module available for Odoo 16 in the OCA e-commerce repository (link: https://github.com/OCA/e-commerce/tree/16.0/website_sale_checkout_skip_payment).

I was wondering if there are any plans to update this module for Odoo 17. This functionality would be very useful for our project, and I would appreciate it if you could let us know if there's any ongoing work or if there are alternative solutions available for Odoo 17.

Thank you for your time and effort in maintaining this valuable module, and I look forward to hearing from you.

**Comment by tracencode** [comment]
Hello Team,

We are working on upgrading it to 17.0 and soon will create a pull request for the same.


Thanks & Regards
Satish S
Manager

TraceNcode Technologies (P) Ltd.
Odoo Partner | Leading Odoo Implementer in Central India

Turning your Imagination to Solution
Odoo | Python | Django | ERP | CRM Development
www.tracencode.com<http://www.tracencode.com/> | +91-9754708777
USA | Chile | India | Belgium



________________________________

**Comment by khalilsido** [comment]
Hi Thank you everyone for the great work I was wondering if this scripts ready yet?

**Comment by pedrobaeza** [comment]
Check the progress on the main comment.

**Comment by asr-informatique** [comment]
Hi, I'd like to know more about the migration of hr_fleet.
I just saw on [hr_fleet/17.0.1.0/upgrade_analysis.txt](https://github.com/OCA/OpenUpgrade/blob/17.0/openupgrade_scripts/scripts/hr_fleet/17.0.1.0/upgrade_analysis.txt) that there is no need to migrate this addon.

Can you mark as "Nothing to do" ?

Thank you

**Comment by dberardo-com** [comment]
hey do you have any ETA on sight ? 

by the way is it possible to use this upgrade script on a kubernetes/postgres installation ? is it as straight forward as doing something like

PGPASSWORD=... PGHOSTADDR=... PGUSER=... python <(curl -s https://upgrade.odoo.com/upgrade) production -d bitnami_odoo -t 17

?

**Comment by luisDIXMIT** [comment]
Hi @legalsylvain I have a doubt. You mentioned that **purchase_price_diff** was merged into purchase_stock module but it isn't declarated on apriori.py file. Is it correct? Same with **account_sequence**.
